app/models/katello/module_stream.rb

```diff
@@ -7,8 +7,16 @@ class ModuleStream < ApplicationRecord
     has_many :profiles, class_name: "Katello::ModuleProfile", dependent: :destroy
     has_many :artifacts, class_name: "Katello::ModuleStreamArtifact", dependent: :destroy
 
-    CONTENT_TYPE = Pulp::ModuleStream::CONTENT_TYPE
+    scoped_search on: :name, complete_value: true
+    scoped_search on: :uuid, complete_value: true
+    scoped_search on: :stream, complete_value: true
+    scoped_search on: :version, complete_value: true
+    scoped_search on: :context, complete_value: true
+    scoped_search on: :arch, complete_value: true
+    scoped_search relation: :repositories, on: :name, rename: :repository, complete_value: true
+    scoped_search relation: :repositories, on: :id, rename: :repository_id, complete_value: true
```

Comment:
does repo id really need to be searchable?

Reply:
I think he added it for https://github.com/Katello/katello/pull/7688/files#diff-2e9419837673072faf0f32caead4e244R137 
May be I could take it out in https://github.com/Katello/katello/pull/7698